src/instructlab/model/download.py

```diff
@@ -350,9 +348,11 @@ def check_skopeo_version():
         if version.parse(installed_version) < version.parse(
             _RECOMMENDED_SCOPEO_VERSION
         ):
-            logger.warning(
-                f"skopeo version {installed_version} is lower than {_RECOMMENDED_SCOPEO_VERSION}. Consider upgrading. Downloading the model might fail."
-            )
+            raise ValueError(f"skopeo version {installed_version} is lower than {_RECOMMENDED_SCOPEO_VERSION}. Consider upgrading.")
+            # click.secho(
+            #     f"skopeo version {installed_version} is lower than {_RECOMMENDED_SCOPEO_VERSION}. Consider upgrading. Downloading the model might fail."
+            # )
+            # raise click.exceptions.Exit(1)
```

Comment:
leftover?

Reply:
oops, thought i removed that before pushing
will remove, thanks!